test/spec/MfaVerify_spec.js

```diff
@@ -342,15 +334,16 @@ function (Q, _, $, Duo, OktaAuth, LoginUtil, CryptoUtil, CookieUtil, Util, MfaVe
             expect(test.form.autoPushCheckbox().length).toBe(0);
           });
         });
-        itp('an answer field type is "password" initially and changed to text \
-          when a "show answer" checkbox is checked', function () {
+        itp('an answer field type is "password" initially and can be switched between "text" and "password" \
+          by clicking on "show"/"hide" buttons', function () {
           return setupSecurityQuestion().then(function (test) {
             var answer = test.form.answerField();
-            expect(test.form.showAnswerCheckboxStatus()).toEqual('unchecked');
             expect(answer.attr('type')).toEqual('password');
-            test.form.setShowAnswer(true);
+
+            test.form.showAnswerButton().click();
```

Comment:
Additional test added.
I can only check the `password-toggle-on` class. Visible or not can be checked only with loaded css, that is not appropriate for tests.